src/components/MapView/index.tsx

```diff
@@ -427,6 +427,14 @@ function MapView({ classes }: MapViewProps) {
           dispatch(setLoadingLayerIds([...layerIds]));
         }
       }
+    };
+    map.on('dataloading', listener);
```

Comment:
Can you explain this new changes?

Reply:
@reed1 - please add some text explaining that last change, then we we can merge. thanks!